src/shared/dissect-image.c

```diff
@@ -2102,9 +2102,14 @@ int dissected_image_mount(
                                 if (r < 0)
                                         return r;
                                 if (r == 0) {
-                                        r = path_is_extension_tree(IMAGE_SYSEXT, where, m->image_name, FLAGS_SET(flags, DISSECT_IMAGE_RELAX_SYSEXT_CHECK));
+                                        r = path_is_extension_tree(IMAGE_SYSEXT, where, m->image_name, FLAGS_SET(flags, DISSECT_IMAGE_RELAX_EXTENSION_CHECK));
                                         if (r < 0)
                                                 return r;
+                                        if (r == 0) {
```

Comment:
check for enoent

Reply:
the path_is_extension_tree function does not return enoent, it returns 0 if nothing is found. Hence that check there..

Reply:
ok